app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Hello, the posts will be listed here ...</h1>
\ No newline at end of file
+<%= render 'shared/header'%>
+<%= render 'shared/user-card', user: @user %>
+<div class = 'posts-container'><%@user.recent_three_posts.each do |post|%>
```

Comment:
- [ ] from the design and project requirement, posts should display the 5 most recent comment, currently your app is displaying all comments here, kindly fix this
design :point_down: 
![Screenshot from 2023-04-06 17-35-52](https://github.com/ShiroYaksha90/blog-app/assets/66097371/de612862-7a2c-42a7-810e-1564ca73d897)
your design :point_down: 
![Screenshot from 2023-07-22 02-08-21](https://github.com/ShiroYaksha90/blog-app/assets/66097371/98444806-88e8-4b98-bf6f-ffc0544cf2b2)